src/missing-types.d.ts

```diff
@@ -26,8 +26,6 @@ declare module '@higlass/libraries' {
     export * as d3Geo from 'd3-geo';
     export * as d3Scale from 'd3-scale';
     export * as d3Selection from 'd3-selection';
-    // export * as d3Queue from 'd3-queue';
-    // export * as d3Request from 'd3-request';
```

Comment:
Is this intended? Could be removed if not needed.

Reply:
The exports for`d3Queue` and `d3Request` were commented out so I just removed them. Perhaps they're unused in the parts of HiGlass that Gosling uses?